packages/editor-ui/src/views/NodeView.vue

```diff
@@ -510,6 +516,36 @@ export default mixins(
 					});
 				}
 			},
+			orientNodes(workflowNodes: INode[]) {
+				const nodes = workflowNodes;
```

Comment:
to avoid mutating original input
```suggestion
			orientNodes(workflowNodes: INode[]) {
				const nodes = [...workflowNodes];
```